### PR TITLE
Feature/translate french

### PR DESCRIPTION
**Pull request by LQ-Sebastien**
I just recreated a link between some lines of text, in the middle of it I didn’t understand what to do, until I realized a translation error 😊

### CODE REVIEW

**Review by rem02 — APPROVED** [review]
Ok for me ^^
Thanks

**Review by rem02 — APPROVED** [review]

File: Translations/French.csv [hunk 1]
```diff
@@ -2165,7 +2165,7 @@ obj.materials.gears,Engrenages,,
 obj.materials.gears.desc,Divers engrenages pour fabriquer des pièces mobiles,,
 obj.materials.glass,Verre,,
 obj.materials.glass.desc,Une matière première précieuse qui est nécessaire pour produire certains équipements. Aussi pour les fenêtres et les sols avec une vue imprenable !,,
-obj.materials.grain,Grain,,
+obj.materials.grain,Grain,,et
```
Comment (rem02) [inline Translations/French.csv:2168]: Why ?

File: Translations/French.csv [hunk 2]
```diff
@@ -1748,7 +1748,7 @@ material.haul,Matériel de transport,,
 material.Iron,Fer,,
 material.lack,Pas assez de {0},,
 material.Lithium,Lithium,,
-material.Log,Enregistrer,,
+material.Log,Buche,,
```
Comment (rem02) [inline Translations/French.csv:1751]: Ok, but like that "Bûche"

File: Translations/French.csv [hunk 3]
```diff
@@ -2171,7 +2171,7 @@ obj.materials.iron,Fer,,
 obj.materials.iron.desc,Le minerai de fer est principalement utilisé pour produire des plaques d'acier.,,
 obj.materials.lithium,Lithium,,
 obj.materials.lithium.desc,Ingrédient essentiel pour les batteries.,,
-obj.materials.log,Enregistrer,,
+obj.materials.log,Buche,,
```
Comment (rem02) [inline Translations/French.csv:2174]: Ok but like that "Bûche"

**Review by rem02 — APPROVED** [review]

File: Translations/French.csv (hunk 1 above)
Comment (LQ-Sebastien) [reply Translations/French.csv:2168]: Oups, it's a mistake

File: Translations/French.csv (hunk 3 above)
Comment (LQ-Sebastien) [reply Translations/French.csv:2174]: Oh damn ! yes

**Review by rem02 — APPROVED** [review]

**Review by rem02 — APPROVED** [review]

**Comment by rem02** [comment]
@LQ-Sebastien Thanks for your time to polish the translation ^^

**Comment by LQ-Sebastien** [comment]
With pleasure, I love this game from the first days, being able to contribute to the translation makes me happy, and I will continue again ^^

**Pull request merged**